src/static/css/pages/_proposals.scss

```
@@ -107,6 +107,7 @@ body.dashboard{
                 }
                 .simple-field, .text-field, .markdown-field {
                     line-height: 150%;
+									  overflow-wrap: anywhere;
```

Please replace the tab with space for consistency.

```suggestion
                    overflow-wrap: anywhere;
```